src/ontology/cl-edit.owl

```diff
@@ -29496,6 +29545,15 @@ AnnotationAssertion(rdfs:label obo:CL_4023129 "retinoblast")
 SubClassOf(obo:CL_4023129 obo:CL_0009004)
 SubClassOf(obo:CL_4023129 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000966))
 
+# Class: obo:CL_4023130 (kisspeptin neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30042117"^^xsd:string) obo:IAO_0000115 obo:CL_4023130 "A neuron that expresses kisspeptin. These neurons are predominantly located in the hypothalamus, but also found in other parts of the brain including the hippocampal dentate gyrus.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023130 "http://orcid.org/0000-0001-7258-9596")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023130 "http://orcid.org/0000-0002-0819-0473")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30042117"^^xsd:string) rdfs:comment obo:CL_4023130 "Gonadal sex steroids stimulate kisspeptin neurons in the RP3V, but inhibit kisspeptin neurons in the ARC.")
+AnnotationAssertion(rdfs:label obo:CL_4023130 "kisspeptin neuron")
+EquivalentClasses(obo:CL_4023130 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000009344)))
```

Comment:
The term as is inherits "part of some nervous system". I'm not sure how far the projections of these neurons go, but based on the definitions, should they be part of some _central_ nervous system more specifically?

Reply:
This should change to something more specific once adding UBERON 'has soma location' comes in. It's not wrong at the moment, so I'd rather not add something that I would have to remove as a redundant subclass later on (cause I will 100% forget to lol)